### PR TITLE
Add all xmb menu options to rgui and glui, 3ds warning fix, move start directory to top of load content

### PR DESCRIPTION
**Pull request by meepingsnesroms**
This preserves all the non emulator entries while allowing all menu options that can be disabled in xmb to be disabled in rgui and possibly glui(all the switches are present for all drivers).

### CODE REVIEW

**Comment by meepingsnesroms** [comment]
It builds fine travis ci just couldnt load certain compilers so it gave an error.

**Comment by orbea** [comment]
I think travis ran out of space...

**Comment by meepingsnesroms** [comment]
Ok, all menu drivers are working with all options other than hide settings and settings password.

File: menu/drivers/materialui.c [hunk 1]
```diff
@@ -36,9 +36,8 @@
 #include "../../config.h"
 #endif
 
-#ifndef HAVE_DYNAMIC
+//this is required regardless of HAVE_DYNAMIC for frontend_driver_parse_drive_list
```
Comment (twinaphex) [inline menu/drivers/materialui.c]: Needs to be a C comment, not C++.

**Pull request merged**